reference/promotions.v3.yml

```diff
@@ -819,6 +819,34 @@ components:
                     minimum: 1
           required:
             - zone_ids
+    ShippingDiscountAction:
+      title: Shipping Discount Action
+      type: object
+      description: |-
+        **Shipping Discount Action**
+        Discounts shipping, optionally restricted to specific shipping methods.
+      properties:
+        shipping_discount:
+          type: object
+          properties:
+            discount:
+              $ref: '#/components/schemas/Discount'
+            method_ids:
+              description: ''
+              oneOf:
+                - type: string
+                  enum:
+                    - '*'
+                  description: All configured shipping methods.
+                - type: array
+                  uniqueItems: true
+                  description: List of shipping method IDs to which the shipping discount can apply.
+                  items:
+                    type: integer
+                    minimum: 1
```

Comment:
Do we want to enforce some sort of sanity check maximum? Just so we don't inevitably have merchants finding creative ways to abuse this and potentially listing hundreds of shipping methods? We can always increase the limit if discover a legit use case

Reply:
Absolutely!